src/components/Additempage/Additempage.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Additemcomponent from './Additemcomponent';
+
+function Additempage() {
```

Comment:
- [ ] Kindly, refactor this function to ES6 arrow function.

Refactoring is easy:

```javascript
const Additempage = () => {
  // ...
}
```

So your code meets the latest standards. 👍